src/Instance.ts

```diff
@@ -177,6 +178,17 @@ export function updateTabsterByAttribute(
                 }
                 break;
 
+            case "restorer":
```

Comment:
Seems like the dispose part is missing (it's in the same file).  And for the sake of consistency, let's do `if (__DEV__)` with the error when Restorer is not initialized yet.

Reply:
Oops. I have missed line 90.